crates/bws/src/state.rs

```diff
@@ -1,17 +1,27 @@
 use std::path::PathBuf;
 
-use color_eyre::eyre::Result;
+use color_eyre::eyre::{bail, Result};
+use directories::BaseDirs;
+
+pub(crate) const DEFAULT_STATE_DIRECTORY: &str = ".config/bws/state";
```

Comment:
Why not Path join `state` with `DEFAULT_CONFIG_DIRECTORY` from `crates/bws/src/cli.rs`, instead of having the `.config/bws` path defined in two places ?

Reply:
Good idea, updated in: https://github.com/bitwarden/sdk/pull/930/commits/1b30f4ee65247e6cd489884c19c9222825cec7a9